src/test/regress/bin/normalize.sed

```diff
@@ -56,9 +56,6 @@ s/\(col_1\)=\([0-9]+\)/(col_1)=(X)/g
 # In multi_name_lengths, normalize shard names
 s/name_len_12345678901234567890123456789012345678_fcd8ab6f_[0-9]+/name_len_12345678901234567890123456789012345678_fcd8ab6f_xxxxx/g
 
-# ignore WAL warnings
-/DEBUG: .+creating and filling new WAL file/d
```

Comment:
I remember this one. It can randomly appear pretty much anywhere. As long as DEBUG logs are on (which we sometimes do). I think it's best to keep it in here.